routes/posts.js

```diff
@@ -0,0 +1,45 @@
+const db = require("/database/model.js");
+
+function get(request, response) {
+try {
+    const postsForm = /*html*/
+`
+    <form enctype="multipart/form-data class="Upload File" id="upload-file" action="/post" method="POST">
+    <div>
+      <label for="Title">Title:</label>
+      <input type="text" id="title" placeholder="Title" name="title" >
+      </div>
+      <br>
+      <label for="alt-text">Alt Text:</label>
+      <input type="text" id="alt_text" placeholder="Alt Text" name="alt_text" >
+      </div>
+      <br>  
+      <div>
+      <label for="file">Upload File:</label>
+      <input type="file" id="file" placeholder="Upload File" name="file" >
+      </div>
+      <br>  
+      <button class="link-as-button submit-button" type="Submit" value="Submit" aria-label="Submit Your Post">Submit</button>
+    </form>
+    `;
+    response.send(postsForm);
+} catch (error) {
+    console.error(error);
+    response.status(500).send(`<h1>Error</h1>`)
+}
+}
+
+    function post(request, response) {
+        const {title, alt_text, image} = sanitize(request.body);
+        model
```

Comment:
I think around here we need to get the users.id to insert into posts as user_id. I think we need the cookies to do this, which aren't set up yet, but maybe worth flagging with a commented line